src/schemas/ExportAccountDataSchema.ts

```
@@ -76,24 +76,24 @@ class ExportAccountDataSchema extends QRCodeDataSchema {
+      throw new Error('JSON argument cannot be empty.');
     }
 
     const jsonObj = JSON.parse(json);
     if (!jsonObj.type || jsonObj.type !== QRCodeType.ExportAccount) {
-      throw new Error("Invalid type field value for AccountQR.");
+      throw new Error('Invalid type field value for AccountQR.');
     }
 
-    if (!jsonObj.hasOwnProperty("data")) {
-      throw new Error("Missing mandatory property for payload.");
+    if (!jsonObj.hasOwnProperty('data')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)